.github/workflows/release.yml

```diff
@@ -39,20 +39,9 @@ jobs:
       with:
         subject-path: "${{ github.workspace }}/${{ env.RELEASE_PACKAGES }}"
 
-    - name: Prepare Nuget
-      uses: hashicorp/vault-action@v3.0.0
-      with:
-        url: ${{ secrets.VAULT_ADDR }}
-        method: approle
-        roleId: ${{ secrets.VAULT_ROLE_ID }}
-        secretId: ${{ secrets.VAULT_SECRET_ID }}
-        secrets: |
-          secret/apm-team/ci/elastic-observability-nuget apiKey | REPO_API_KEY ;
-          secret/apm-team/ci/elastic-observability-nuget url | REPO_API_URL
-
     - name: Release to Nuget (only for release events)
       if: ${{ github.event_name == 'release' }}
-      run: dotnet nuget push '${{ env.RELEASE_PACKAGES }}' -k ${{secrets.NUGET_REPO_API_KEY}} -s https://api.nuget.org/v3/index.json --skip-duplicate --no-symbols
+      run: dotnet nuget push '${{ env.RELEASE_PACKAGES }}' -k ${{ secrets.NUGET_API_KEY }} -s https://api.nuget.org/v3/index.json --skip-duplicate --no-symbols
```

Comment:
I don't know who created `secrets.NUGET_REPO_API_KEY` but `${{ secrets.NUGET_API_KEY }}` is the one I'm now using it.

Rationale, we are tracking with some automation what GitHub secrets we create and we use Backstage to track them down easily.